app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -305,6 +304,16 @@ def handle_expired_status_update(enrollment, response, response_message)
     end
   end
 
+  # MW: What else do we want to log here?
+  def handle_fraud_review_pending(enrollment)
```

Comment:
Should this be behind the flag Keith is setting in his issue? You mentioned waiting on testing until that work is complete but just wanted to make sure we add that additional conditional in.

Reply:
Yes, thank you for flagging.

Reply:
what does `MW` stand for?

Reply:
oh wait I'm guessing your initials based on a comment later on in this pr

Reply:
Yes, I usually use that or `mattw:` for little notes to myself.

I suppose `TODO:` or `FIXME:` would actually be better to ensure they get removed before merge, since it'll fail the linter.

Reply:
Feature flag added! Resolving this comment.